src/Goerli-DssSpell.sol

```diff
@@ -46,19 +134,438 @@ contract DssSpellAction is DssAction {
+            DOC:             RWA010_A_DOC
+        }));
+
+
+        // RWA-013 Onboarding
+        // https://vote.makerdao.com/polling/QmNucsGt
+        _addCentrifugeCollateral(CentrifugeCollateralValues({
+            GEM_JOIN:        MCD_JOIN_RWA013_A,
+            GEM:             RWA013,
+            URN:             RWA013_A_URN,
+            OPERATOR:        RWA013_A_OPERATOR,
+            INPUT_CONDUIT:   RWA013_A_INPUT_CONDUIT,
+            OUTPUT_CONDUIT:  RWA013_A_OUTPUT_CONDUIT,
+            DROP:            0x0691FAEa2Eb8eBB2C36Fc24d577cA73AfbDB7Bdd,
+            OWNER:           0x116b030167Cc8A82C158f0598f4C4677f575Cc50, // Tinlake Clerk
+            POOL:            0x3C05eFC8D0fC042c5686b3989bbDb1E1D29dAec7, // Tinlake Operator
+            TRANCHE:         0x6A635Ada2eC663B9b38ca7a3E5c918D5D0B0E99D, // Tinlake Tranche
+            gemID:           "RWA013",
+            joinID:          "MCD_JOIN_RWA013_A",
+            urnID:           "RWA013_A_URN",
+            inputConduitID:  "RWA013_A_INPUT_CONDUIT",
+            outputConduitID: "RWA013_A_OUTPUT_CONDUIT",
+            pipID:           "PIP_RWA013",
+            ilk:             "RWA013-A",
+            ilkString:       "RWA013",
+            ilkRegistryName: "RWA013-A: Centrifuge: BlockTower Credit (IV)",
+            RATE:            FOUR_PCT_RATE,
+            CEIL:            70_000_000,
+            PRICE:           85_165_703 * WAD,
+            MAT:             100_00, // Liquidation ratio
+            TAU:             0,      // Remediation period
+            DOC:             RWA013_A_DOC
+        }));
+
+
+        // ----------------------------- Collateral offboarding -----------------------------
+        //  Offboard RENBTC-A
+        //  Poll Link:   https://vote.makerdao.com/polling/QmTNMDfb#poll-detail
+        //  Forum Post:  https://forum.makerdao.com/t/renbtc-a-proposed-offboarding-parameters-context/18864
+
+        DssExecLib.setIlkLiquidationPenalty("RENBTC-A", 0);
+        DssExecLib.setKeeperIncentiveFlatRate("RENBTC-A", 0);
+        // setIlkLiquidationRatio to 5000%
+        // We are using low level methods because DssExecLib only allows setting `mat < 1000%`: https://github.com/makerdao/dss-exec-lib/blob/2afff4373e8a827659df28f6d349feb25f073e59/src/DssExecLib.sol#L733
+        DssExecLib.setValue(DssExecLib.spotter(), "RENBTC-A", "mat", 50 * RAY); // 5000%
+        DssExecLib.setIlkMaxLiquidationAmount("RENBTC-A", 350_000);
+        // PIP_RENBTC `kiss` MCD_CLIP_RENBTC_A. This should not be included in mainnet spell
+        DssExecLib.addReaderToWhitelist(DssExecLib.getChangelogAddress("PIP_RENBTC"), DssExecLib.getChangelogAddress("MCD_CLIP_RENBTC_A"));
+
+        // Increase Starknet Bridge Limit from 200,000 DAI to 1,000,000 DAI
+        StarknetLike(STARKNET_DAI_BRIDGE).setCeiling(1_000_000 * WAD);
+        // Remove Starknet Bridge Deposit Limit
+        StarknetLike(STARKNET_DAI_BRIDGE).setMaxDeposit(type(uint256).max);
```

Comment:
Starknet part of the spell changes two bridge parameters related to deposits:
* *bridge ceiling* (`setCeiling`) - total amount that can be bridged
* *deposit limit* (`setMaxDeposit`) - max amount that can be bridged by single deposit
Initiality, when bridge ceiling was small, `deposit limit` prevented few large users from using whole bridge ceiling.
Since now we are rising bridge ceiling to relatively large number (for Starknet) it makes sense to remove the deposit alltogether.